examples/JsonSerialization/Program.cs

```diff
@@ -149,11 +149,16 @@ static async Task Main(string[] args)
                 while ((text = Console.ReadLine()) != "q")
                 {
                     User user = new User { Name = text, FavoriteColor = "blue", FavoriteNumber = i++ };
-                    await producer
-                        .ProduceAsync(topicName, new Message<string, User> { Value = user })
-                        .ContinueWith(task => task.IsFaulted
-                            ? $"error producing message: {task.Exception.Message}"
-                            : $"produced to: {task.Result.TopicPartitionOffset}");
+                    var task = producer.ProduceAsync(topicName, new Message<string, User> { Value = user });
+                    task.Wait();
```

Comment:
get rid of everything except the ProduceAsync call, and await the ProduceAync call.